nmrfx-utils/src/main/java/org/nmrfx/utils/properties/ListOperationItem.java

```diff
@@ -120,44 +110,50 @@ public void setValue(Object o) {
                 for (String sValue : lst.split(",")) {
                     switch (sValue) {
                         case "":
-                            newValue.add(0.0);
+                            numberValues.add(0.0);
                             break;
                         case "-":
-                            newValue.add(-0.0);
+                            numberValues.add(-0.0);
                             break;
                         default:
                             try {
-                                newValue.add(Double.parseDouble(sValue));
+                                numberValues.add(Double.parseDouble(sValue));
                             } catch (NumberFormatException nfE) {
-                                newValue.add(0.0);
+                                numberValues.add(0.0);
                             }
                             break;
                     }
                 }
             }
+            newValue = numberValues;
         } else if (o instanceof ArrayList) {
-            newValue = new ArrayList((ArrayList) o);
+            newValue = new ArrayList<>((ArrayList<?>) o);
         } else {
             return;
         }
```

Comment:
Value was never set unless a string or an array list was passed in, left this functionality as is.

Reply:
Which is fine for this PR. 

In general though, I think this pattern is quite bad, and it should throw an `IllegalArgumentException` instead. I'm not sure whether this will break anything with existing code, so letting it as-is for now is ok. 

Maybe changing the instanceof check to accept any `List` type instead of only `ArrayList`, since we create a new one anyway, but that's again increasing this PR scope.